service-front/app/src/Viewer/src/Form/LpaCheck.php

```diff
@@ -71,7 +103,7 @@ public function getInputFilterSpecification(): array
                         'name'                   => Regex::class,
                         'break_chain_on_failure' => true,
                         'options'                => [
-                            'pattern' => '/M[-\s]?(\d{4}[-\s]?){2}\d{4}$/',
+                            'pattern' => '/^[Mm]-\d{4}-\d{4}-\d{4}$/',
```

Comment:
```suggestion
                            'pattern' => '/^M-\d{4}-\d{4}-\d{4}$/',
```

since you are setting the format above, I think